### PR TITLE
feat(slider) - adding slider manager

### PR DESCRIPTION
**Pull request by souhe**
It has all not 'IOS only' props from http://facebook.github.io/react-native/docs/slider.html. The only difference is event handlers argument. For IOS and Android it is only value, here it's whole `event` object with value field. 
#373

### CODE REVIEW

File: ReactWindows/ReactNative/Views/Slider/ReactSliderManager.cs [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+        /// <summary>
+        /// Subclasses can override this method to install custom event 
+        /// emitters on the given view.
+        /// </summary>
+        /// <param name="reactContext">The React context.</param>
+        /// <param name="view">The view instance.</param>
+        protected override void AddEventEmitters(ThemedReactContext reactContext, Windows.UI.Xaml.Controls.Slider view)
+        {
+            view.ValueChanged += OnValueChange;
+            view.PointerReleased += OnPointerReleased;
```
Comment (rozele) [inline ReactWindows/ReactNative/Views/Slider/ReactSliderManager.cs]: Just curious if these are the correct events.  Could we just use view.LostFocus or does it have to be fired after the pointer is released?

Comment (rozele) [inline ReactWindows/ReactNative/Views/Slider/ReactSliderManager.cs]: Nevermind just looked at the Android implementation, looks like they use touch as well.

---
In reply to: [76804165](https://github.com/ReactWindows/react-native-windows/pull/655#discussion_r76804165) [](ancestors = 76804165)

**Comment by rozele** [comment]
I was getting a crashing bug every time I ran the UIExplorer Slider example.  Have you run this view manager in the UIExplorer yet?

**Comment by souhe** [comment]
Oh, I was testing it in Playground... but you're right. I'll fix it!

**Comment by souhe** [comment]
The problem was with default 0 value in `step` prop. UWP Slider requires value different than 0. Above commit should fix this issue.

**Pull request merged**